doc/readme.md

```diff
@@ -1,7 +1,7 @@
 ---
 home: true
 heroImage: /img/gerbil-detailed-consio_OPT.svg
-tagline: Gerbil is a meta-dialect of Scheme with post-modern features
+tagline: A LISP for the 21st Century
```

Comment:
LISP is a very 1950s to early 1980s spelling. Maybe make it a Lisp for the 21st Century?

Reply:
It is the original spelling, I love it.
For me, the abstract language of languages has always been capital spelled.